src/smt/solver_engine_state.cpp

```diff
@@ -118,9 +118,9 @@ void SolverEngineState::notifyCheckSatResult(bool hasAssumptions, Result r)
   }
 }
 
-void SolverEngineState::notifyCheckSynthResult(Result r)
+void SolverEngineState::notifyCheckSynthResult(SynthResult r)
```

Comment:
Shouldn't this not be passed by value but by const reference?

Reply:
Yes it can be.